NdagiStanley/vue-django
# VueJS 2 & Django 2 upgrade

#### What does this PR do?
- Updates the Dockerfile and the entire project to run in VueJS 2.x and Django 2.x
- Open up the project for Open Source contribution
- Give options for different versions of `Django` and `VueJS`
#### Description of Task to be completed?
- Update Dockerfile to `node:alpine` and compatibility with `Django 2.x` which only works with `Python 3.x`
- Add `codetriage` badge to README
- Add reference to branches #v1-d1 and #v2-d1 in the READMEs at the root of the project and Docs directory
#### How should this be manually tested?
- Build and run the images in either of the three branches `develop`, `v1-d1` and `v2-d1` by running: 
```bash
docker-compose up --build
```
- Go to the [github page](https://ndagistanley.github.io/vue-django/) and confirm the options are laid out well.
#### Any background context you want to provide?
Fixes issue #74
